app/connector/google-sheets/src/main/resources/META-INF/syndesis/connector/sheets.json

```diff
@@ -820,18 +806,97 @@
       ],
       "type": "hidden"
     },
+    "googleScopes": {
+      "deprecated": false,
+      "displayName": "Scopes",
+      "group": "common",
+      "javaType": "java.lang.String",
+      "kind": "parameter",
+      "labelHint": "UserId",
+      "raw": true,
+      "required": true,
+      "secret": false,
+      "tags": [
+        "oauth-scope"
+      ],
+      "type": "hidden"
+    },
     "userId": {
       "deprecated": false,
       "displayName": "User ID",
       "group": "common",
       "javaType": "java.lang.String",
       "kind": "parameter",
       "labelHint": "Google Sheets account name that is associated with this registration.",
-      "order": "6",
+      "order": "7",
+      "raw": true,
+      "required": false,
+      "secret": false,
+      "type": "string"
+    },
+    "additionalOAuthQueryParameters": {
+      "deprecated": false,
+      "displayName": "Additional OAuth query parameters",
+      "group": "security",
+      "javaType": "java.lang.String",
+      "kind": "parameter",
       "raw": true,
       "required": true,
       "secret": false,
+      "tags": [
+        "oauth-additional-query-parameters"
+      ],
+      "type": "hidden"
+    },
+    "validateCertificates": {
```

Comment:
perhaps we would like to prevent users from changing this (make it `hidden`)?

Reply:
Yeah, this feels a bit ugly. But how would a tester then be able to set this property when using a simulator deployed to Openshift in the test environment?